src/main/webapp/components/CheckboxGroup/checkboxGroupMocks.ts

```diff
@@ -0,0 +1,309 @@
+                                            'collectFileMetadata',
+                                            'collectTasksAndComments',
+                                        ],
+                                    },
+                                    {
+                                        label: 'Collect2',
+                                        options: {
+                                            isExpandable: true,
+                                        },
+                                        fields: ['collectFolderMetadata'],
+                                    },
+                                ],
+                                rows: [
+                                    {
+                                        field: 'collectFolderCollaboration',
+                                        text: {
+                                            defaultValue: 1200,
+                                            required: false,
+                                            validators: [
+                                                {
+                                                    type: 'number',
+                                                    range: [1, 1200],
+                                                },
+                                            ],
+                                        },
+                                    },
+                                    {
+                                        field: 'collectFileMetadata',
+                                        checkbox: {
+                                            label: 'Collect file metadata',
+                                        },
+                                        text: {
+                                            defaultValue: 1,
+                                            required: true,
+                                        },
+                                    },
+                                    {
+                                        field: 'collectTasksAndComments',
+                                        checkbox: {
+                                            label: 'This is a very very long line',
+                                            options: {
+                                                enable: true,
+                                            },
+                                        },
+                                        text: {
+                                            defaultValue: 1,
+                                            required: true,
+                                        },
+                                    },
+                                    {
+                                        field: 'collectFolderMetadata',
+                                        checkbox: {
+                                            label: 'Collect folder metadata',
+                                            defaultValue: 0,
```

Comment:
`defaultValue` is not allowed here according to the schema, @vtsvetkov-splunk can you please check?

Reply:
yeah, unfortunately I have not found the great way for auto checking whatever our code models are sync with schema.json. I will fix it

Reply:
Resolved in #441